src/applications/edu-benefits/0993/helpers.js

```diff
@@ -24,3 +25,14 @@ export function transform(formConfig, form) {
     }
   });
 }
+/* eslint-disable */
+export const tabbableTitle = title => (<div
+  className="schemaform-block-header"
+  tabIndex="0">
+  <p
+    className="schemaform-block-title schemaform-block-subtitle"
+    id="root_view:optOutMessage__title">
```

Comment:
Is there an extra underscore here?

Reply:
TBH I think we just copied what was on the generated site verbatim. Probably can clean this up- no reason to follow the generated pattern.